data.json

```diff
@@ -6099,7 +6102,7 @@
                 "adobe-flashplayer",
                 "AdobeFlash",
                 "adobeflashplugin",
-                "com.Adobe.Flash-Player-Projector",
+                "com.adobe.Flash-Player-Projector",
```

Comment:
Is there a reason this is a rename rather than a new icon?

Reply:
It's not an addition, the old one was a typo.